test/user/signup.js

```diff
@@ -105,15 +105,95 @@ describe('Signup API', () => {
       .set('Content-Type', 'application/json')
       .send({
         name: 'Full name',
-        email: 'testtt@test.com',
-        username: 'user',
+        email: 'tester@test.com',
+        username: 'user_test',
         password: 'password'
       })
       .end((err, res) => {
+        // console.log(res);
         expect(res.status).to.equal(201);
-        expect(err).to.be.null;
         expect(res.body).to.be.an('object');
         done();
       });
   });
 });
+
+describe('Signin', () => {
+  it('should not allow a user to login with no email', (done) => {
```

Comment:
I thought these test were for signup? I would expect the test description to match.

Reply:
Thanks for your feedback. The test descriptions and file name have been refactored to match.